app/lib/actions/katello/repository_set/enable_repository.rb

```diff
@@ -2,6 +2,8 @@ module Actions
   module Katello
     module RepositorySet
       class EnableRepository < Actions::EntryAction
+        middleware.use Actions::Middleware::BackendServicesCheck
```

Comment:
That is not a bad idea.  It would cover every single action which i wasn't originally attempting to do.  But i guess we could do that